### PR TITLE
Locust test harness to step through blue-green

### PR DESCRIPTION
**Pull request by mjmenger**

### CODE REVIEW

**Review by jtylershaw — APPROVED** [review]

File: fast/test/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```python
+VIP_INFO = [
+    ("192.0.2.10","Test","App"),
+    ("192.0.2.11","Test2","App")
+]
+```
+
+## Start a Locust Instance
+at a command line the following environment variables
```
Comment (aknot242) [inline fast/test/README.md:34]: create? add?

Comment (aknot242) [reply fast/test/README.md:34]: How about "at a command line, DEFINE the following environment variables"

**Review by aknot242 — CHANGES_REQUESTED** [review]
Just one comment change suggestion inline.

File: fast/test/README.md (hunk 1 above)
Comment (aknot242) [reply fast/test/README.md:34]: Whoops, I was too late. :)

Comment (mjmenger) [reply fast/test/README.md:34]: will do

**Pull request merged**